gradle/dependencies.gradle

```diff
@@ -1,5 +1,6 @@
 def versions = [
-    androidPlugin       : '4.2.2',
+    androidPlugin       : '7.0.0',
```

Comment:
This version bump is actually not required if using toolchains but since IntelliJ now supports 7.0.0 it shouldn't hurt to bump.